__tests__/app.test.js

```diff
@@ -182,6 +182,25 @@ describe("GET - Endpoints", () => {
         });
     });
   });
+  describe('GET /api/users', () => {
+    test('200: Responds with an array of user objects, each having username, name, and avatar_url', () => {
+      return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.users).toBeInstanceOf(Array);
+          body.users.forEach((user) => {
```

Comment:
It's important to check the length of the array first before using a forEach. The reason is that if your array was empty, the forEach is skipped and all your assertions inside it will be skipped.